internal/catalog/internal/controllers/failover/controller_test.go

```diff
@@ -29,18 +31,22 @@ type controllerSuite struct {
 	failoverMapper FailoverMapper
 
 	ctl failoverPolicyReconciler
+
+	tenancies []*pbresource.Tenancy
 }
 
 func (suite *controllerSuite) SetupTest() {
 	suite.ctx = testutil.TestContext(suite.T())
-	client := svctest.RunResourceService(suite.T(), types.Register)
+	client := svctest.RunResourceServiceWithTenancies(suite.T(), types.Register, types.RegisterDNSPolicy)
 	suite.rt = controller.Runtime{
 		Client: client,
 		Logger: testutil.Logger(suite.T()),
 	}
 	suite.client = rtest.NewClient(client)
 
 	suite.failoverMapper = failovermapper.New()
+
```

Comment:
It looks like this doesn't quite match the ENT PR. One difference I noticed is the `suite.isEnterprise` being set there, but not here.

Could you ensure the PRs are in sync?